games-sports/speed-dreams/speed_dreams-2.2.3.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Open Motorsport Simulator"
+DESCRIPTION="Speed Dreams is a 3d cross-platform, open source motorsport simulation and racing game.
+
+Speed Dreams is a fork of the open racing car simulator Torcs, aiming to implement exciting new \
+features, cars, tracks and AI opponents to make a more enjoyable game for the player, as well as \
+constantly improving visual and physics realism."
+HOMEPAGE="http://www.speed-dreams.org/"
+COPYRIGHT="2009-2018 Jean-Philippe Meuret"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://sourceforge.net/projects/speed-dreams/files/$portVersion/speed-dreams-src-base-$portVersion-r7616.tar.xz"
+CHECKSUM_SHA256="c67e48a083cd86f6c0062f0b38ed91a5d25274ea8aa656ee182f3f5074e96ff0"
+SOURCE_DIR=""
+ADDITIONAL_FILES="speed_dreams.rdef.in"
+PATCHES="speed_dreams-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	speed_dreams$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+#	lib:plib$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libenet$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libogg$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+#	lib:libSDL2_2.0$secondaryArchSuffix	
+	lib:libvorbis$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+#	devel:plib$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libenet$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopenal$secondaryArchSuffix
+	devel:libplibnet$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libsdl$secondaryArchSuffix
```

Comment:
```diff
-	devel:libpng$secondaryArchSuffix
-	devel:libsdl$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libSDL_1.2$secondaryArchSuffix
```

Reply:
Issue for those are mentioned in: https://github.com/haikuports/haikuports/issues/3878 and https://github.com/haikuports/haikuports/issues/3779